src/main/groovy/com/blackbaud/templates/tasks/RestProject.groovy

```diff
@@ -198,7 +225,7 @@ import ${servicePackage}.client.${resourceName}Client;
 
     @Bean
     public ${resourceName}Client ${resourceNameLowerCamel}Client() {
-        return testClientSupport().createClientWithTestToken(${resourceName}Client.builder());
+        return testClientSupport.createClientWithTestToken(${resourceName}Client.class);
```

Comment:
is this test token a long auth token?do we care? i think this is also an example so may not matter...

Reply:
kinda... it's a special type of token which has all the claims of a next gen token.  check out com.blackbaud.security.token.LuminateNextGenTokenProvider#createAccessToken in common-spring-boot